src/LineHandler.mss

```diff
@@ -8,100 +8,100 @@ function InitLineHandlers () {
+        // 'line.staff.letring',                       @Element('Line', @Attrs('form', 'solid')),
+        // 'line.staff.mute.palm',                     @Element('Line', @Attrs('form', 'solid')),
         // TODO: The Nebenstimme line type only has a start symbol and an end symbol without an actual line.
-        'line.staff.nebenstimme',              CreateSparseArray('Line', CreateDictionary('startsym', 'N', 'endsym', 'angledown', 'lwidth', '0', 'endid', 'Previous')),
-        // 'line.staff.pick.scrape',              CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.rake',                     CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.slide',                    CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.1',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.2',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.3',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.4',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.5',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.6',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.7',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.above.8',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.1',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.2',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.3',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.4',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.5',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.6',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.7',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        // 'line.staff.string.below.8',           CreateSparseArray('Line', CreateDictionary('form', 'solid')),
-        'line.staff.vertical',                 CreateSparseArray('Line', CreateDictionary('form', 'solid', 'type', verticalLine, 'endid', 'PreciseMatch')),
+        'line.staff.nebenstimme',                   @Element('Line', @Attrs('startsym', 'N', 'endsym', 'angledown', 'lwidth', '0', 'endid', 'Previous')),
```

Comment:
In MEI 4 it's `width`:
```suggestion
        'line.staff.nebenstimme',                   @Element('Line', @Attrs('startsym', 'N', 'endsym', 'angledown', 'width', '0', 'endid', 'Previous')),
```
If you think this is the right solution, than it has to be also present on `line.staff.hauptstimme`.